litmus-portal/frontend/src/views/Settings/AccountsTab/AccountSettings/index.tsx

```diff
@@ -221,7 +244,11 @@ const AccountSettings: React.FC = () => {
                         isDisabled={false}
                         handleClick={handleClose}
                       >
-                        <>Done</>
+                        <>
+                          {t(
+                            'settings.accountsTab.accountsSettings.button.done'
+                          )}
+                        </>
```

Comment:
You won't require Fragments anymore since children are auto inferred would you like to remove ButtonFilled's `children: React.Node` [Remove children from the interface and replace all the buttons that are wrapped with Fragment to not use Fragment] in this PR or open a separate PR for that change later?

Reply:
I would like to do this in a separate PR. This one already has too many file changes.